index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Inter&family=Poppins:wght@100;200&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+</html>
+<body>
+  <header class="header">
+    <nav class="header_nav">
+      <div class="header_logo black"><a class="black" href="#">Welcome</a></div>
+      <div class="header_menu">
+        <img class="menu_icon" src="images/Union.png" alt="menu" />
+      </div>
+    </nav>
+  </header>
+  <section class="headline">
+    <h1 class="headline_title">
+      <span class="sub-headline_title_1">Hey there. I’m 
+        <br>Cansu</span>
+      <span class="sub-headline_title">I'm a software 
+        <br>developer</span>
+    </h1>
+    <p class="headline_content">
+      I can help you build a product , feature or website Look through some of
+      my work and experience! If you like what you see and have a project you
+      need coded, don’t hestiate to contact me. </p>
+    <div class="backimg"></div>
+    <div class="addline_social">
+      <ul class="social_links">
+        <li class="social_link">
+          <a href="#">
+            <img class="social_icon" src="images/twitter.png" alt="twitter" />
+          </a>
+        </li>
+        <li class="social_links"></li>
+        <li class="social_link">
+          <a href="#">
+            <img
+              class="social_icon"
+              src="images/Linkedin icon.png"
+              alt="linkedin"
+            />
+          </a>
+        </li>
+        <li class="social_link">
+          <a href="#">
+            <img class="social_icon" src="images/github.png" alt="github" />
+          </a>
+        </li>
+        <li class="social_link">
+          <a href="#">
+            <img class="social_icon" src="images/medium.png" alt="M" />
+          </a>
+        </li>
+        <li class="social_link">
+          <a href="#">
+            <img class="social_icon" src="images/angellist.png" alt="Hand" />
+          </a>
```

Comment:
- [ ] Good job @JansueT :+1: , however the link to the hamburger image is broken. Kindly provide a proper url :pray: 

Your design

![Screenshot from 2022-12-02 19-31-46 (1)](https://user-images.githubusercontent.com/80458199/205352335-5cd359b3-422e-4f44-a094-2f4a0a2de8c2.png)